web/shapeworks/src/views/Select.vue

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+import { defineComponent, onMounted } from '@vue/composition-api';
+import { getDatasets } from '@/api/rest'
+import { allDatasets, selectedDataset } from '../store/index';
+import { Dataset } from '@/types';
+import router from '@/router/routes';
+
+export default defineComponent({
+    setup() {
+        onMounted(async () => {
+            selectedDataset.value = undefined;
+            if(allDatasets.value.length === 0){
+                allDatasets.value = await getDatasets();
+            }
```

Comment:
```suggestion
            allDatasets.value = await getDatasets();
```
My logic here is that if a dataset is already selected, then it's confusing if you go the dataset selection page and suddenly all the other tabs disappear because your dataset was automatically deselected.
Also, I don't see a reason to not check for new datasets every time the component is mounted. If nothing has changed the user won't notice anything happening.

Reply:
Good points. Both changed in [e059423](https://github.com/girder/shapeworks-cloud/pull/132/commits/e059423f0ddd274d03af1760f40333ac24bcc9a3)